apps/api/src/utils/resume_handler.py

```diff
@@ -0,0 +1,52 @@
+import hashlib
+import os
+from logging import getLogger
+from typing import Protocol
+
+from aiogoogle import HTTPError
+from fastapi import UploadFile
+
+from services import gdrive_handler
+
+log = getLogger(__name__)
+
+RESUMES_FOLDER_ID = os.getenv("RESUMES_FOLDER_ID")
```

Comment:
Could we add a note to the API README that we need this environment variable (as well as `GOOGLE_SERVICE_ACCOUNT_CREDENTIALS`) for deployment?